GitUI/Editor/FileViewer.cs

```diff
@@ -982,22 +969,10 @@ private void encodingToolStripComboBox_SelectedIndexChanged(object sender, Event
             Encoding encod = null;
             if (string.IsNullOrEmpty(encodingToolStripComboBox.Text))
                 encod = Module.FilesEncoding;
-            else if (encodingToolStripComboBox.Text.StartsWith("Default", StringComparison.CurrentCultureIgnoreCase))
-                encod = Encoding.Default;
```

Comment:
Encoding.Default in list, in .NET not encoding with name "Default", in my system it's Cyrillic(windows). This code determines which object of encoding need use by name. Before editing list filled with name  "Default...". Now list contains this object, but it named different.
Or I not correctly understood?